UPGRADE-3.x.md

```diff
@@ -1,6 +1,16 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.2 to 3.3
```

Comment:
:warning: This **must** not be added on PR, please take care @sonata-project/contributors